src/pages/Index.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <q-page class="column justify-start items-center q-px-sm q-pt-xl">
     <div class="text-h4 text-weight-thin">VOTE FOR YKNOT</div>
-    <div class="subtitle text-center q-my-md">
+    <div class="subtitle text-center q-my-md" style="font-size: 2.5vh">
```

Comment:
- Rather place the `font-size` in the style section of this file, under the `.subtitle` class.
- Change the `font-size` to `1.2rem`. I tested it out and this `rem` value works a bit better. The `vh` is nice, but you have to clamp or min / max it to look good. Too tweaky imo